setup.cfg

```diff
@@ -1,5 +1,107 @@
+    interactively.  Its main components are:
+
+    * A powerful interactive Python shell
+    * A `Jupyter <https://jupyter.org/>`_ kernel to work with Python code in Jupyter
+    notebooks and other interactive frontends.
+
+    The enhanced interactive Python shells have the following main features:
+
+    * Comprehensive object introspection.
+
+    * Input history, persistent across sessions.
+
+    * Caching of output results during a session with automatically generated
+    references.
+
+    * Extensible tab completion, with support by default for completion of python
+    variables and keywords, filenames and function keywords.
+
+    * Extensible system of 'magic' commands for controlling the environment and
+    performing many tasks related either to IPython or the operating system.
+
+    * A rich configuration system with easy switching between different setups
+    (simpler than changing $PYTHONSTARTUP environment variables every time).
+
+    * Session logging and reloading.
+
+    * Extensible syntax processing for special purpose situations.
+
+    * Access to the system shell with user-extensible alias system.
+
+    * Easily embeddable in other Python programs and GUIs.
+
+    * Integrated access to the pdb debugger and the Python profiler.
+
+    The latest development version is always available from IPython's `GitHub
+    site <http://github.com/ipython>`_.
+
 license_file = LICENSE
+project_urls =
+    Documentation = https://ipython.readthedocs.io/
+    Funding = https://numfocus.org/
+    Source = https://github.com/ipython/ipython
+    Tracker = https://github.com/ipython/ipython/issues
+keywords = Interactive, Interpreter, Shell, Embedding
+platforms = Linux, Mac OSX, Windows
+classifiers =
+    Framework :: IPython
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3 :: Only
+    Topic :: System :: Shell
+
+
+[options]
+packages = find:
+python_requires = >=3.7
+zip_safe = False
+install_requires =
+    setuptools>=18.5
+    jedi>=0.16
+    decorator
+    pickleshare
+    traitlets>=4.2
+    prompt_toolkit>=2.0.0,<3.1.0,!=3.0.0,!=3.0.1
+    pygments
+    backcall
+    stack_data
+    matplotlib-inline
+    pexpect>4.3; sys_platform != "win32"
+    appnope; sys_platform == "darwin"
+    colorama; sys_platform == "win32"
+
+[options.packages.find]
+exclude =
+    deathrow
+    quarantine
+    setupext
+
+[options.package_data]
+IPython.core = profile/README*
+IPython.core.tests = *.png, *.jpg, daft_extension/*.py
+IPython.lib.tests = *.wav
+IPython.testing.plugin = *.txt
+
+[options.entry_points]
+console_scripts =
+    ipython = IPython:start_ipython
+    iptest = IPython.testing.iptestcontroller:main
+    ipython3 = IPython:start_ipython
+    iptest3 = IPython.testing.iptestcontroller:main
```

Comment:
iptest was removed on trunk, could you please rebase?

Reply:
Will do.